content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -166,151 +152,174 @@ A Deployment's rollout is triggered if and only if the Deployment's pod template
-Alternatively, you can `edit` the Deployment and change `.spec.template.spec.containers[0].image` from `nginx:1.7.9` to `nginx:1.9.1`:
-
-```shell
-kubectl edit deployment.v1.apps/nginx-deployment
-```
-```
-deployment.apps/nginx-deployment edited
-```
+1. Let's update the nginx Pods to use the `nginx:1.9.1` image instead of the `nginx:1.7.9` image.
 
-To see the rollout status, run:
+    ```shell
+    kubectl --record deployment.apps/nginx-deployment set image deployment.v1.apps/nginx-deployment nginx=nginx:1.9.1
+    ```
+    The output is similar to this:
+    ```
+    deployment.apps/nginx-deployment image updated
+    ```
 
-```shell
-kubectl rollout status deployment.v1.apps/nginx-deployment
-```
-```
-Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
-deployment.apps/nginx-deployment successfully rolled out
-```
+    Alternatively, you can `edit` the Deployment and change `.spec.template.spec.containers[0].image` from `nginx:1.7.9` to `nginx:1.9.1`:
 
-After the rollout succeeds, you may want to `get` the Deployment:
+    ```shell
+    kubectl edit deployment.v1.apps/nginx-deployment
+    ```
 
-```shell
-kubectl get deployments
-```
-```
-NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-nginx-deployment   3         3         3            3           36s
-```
+    The output is similar to this:
+    ```
+    deployment.apps/nginx-deployment edited
+    ```
 
-The number of up-to-date replicas indicates that the Deployment has updated the replicas to the latest configuration.
-The current replicas indicates the total replicas this Deployment manages, and the available replicas indicates the
-number of current replicas that are available.
+2. To see the rollout status, run:
 
-You can run `kubectl get rs` to see that the Deployment updated the Pods by creating a new ReplicaSet and scaling it
-up to 3 replicas, as well as scaling down the old ReplicaSet to 0 replicas.
+    ```shell
+    kubectl rollout status deployment.v1.apps/nginx-deployment
+    ```
 
-```shell
-kubectl get rs
-```
-```
-NAME                          DESIRED   CURRENT   READY   AGE
-nginx-deployment-1564180365   3         3         3       6s
-nginx-deployment-2035384211   0         0         0       36s
-```
+    The output is similar to this:
+    ```
+    Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
+    ```
+    or
+    ```
+    deployment.apps/nginx-deployment successfully rolled out
+    ```
 
-Running `get pods` should now show only the new Pods:
+Get more details on your updated deployment:
 
-```shell
-kubectl get pods
-```
-```
-NAME                                READY     STATUS    RESTARTS   AGE
-nginx-deployment-1564180365-khku8   1/1       Running   0          14s
-nginx-deployment-1564180365-nacti   1/1       Running   0          14s
-nginx-deployment-1564180365-z9gth   1/1       Running   0          14s
-```
+* After the rollout succeeds, you may want to `get` the Deployment:
 
```

Comment:
Instead of `get` the Deployment, 
... you can view the Deployment [by running `kubectl get deployments]